beginner_source/basics/data_tutorial.py

```diff
@@ -44,6 +44,14 @@
 #  - ``download=True`` downloads the data from the internet if it's not available at ``root``.
 #  - ``transform`` and ``target_transform`` specify the feature and label transformations
 
+############################################################
+# Loading a Dataset
+# -------------------
+# Add these 2 lines of code to control the environment, in case Conda is used.
+#
+
+import os
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
```

Comment:
Thanks for the PR @ver2king , but as mentioned in https://github.com/pytorch/tutorials/issues/1692#issuecomment-1582242206 I'm not sure we actually need to do anything. CC @svekars